contracts/V1/StateChannelDiamondProxy/DisputeManagerFacet.sol

```diff
@@ -99,121 +99,168 @@ contract DisputeManagerFacet is StateChannelCommon {
                 .executeStateTransitionOnState(channelId, encodedState, _tx);
     }
 
-    function getDispute(
-        bytes32 channelId
-    ) public view returns (Dispute memory) {
-        return disputes[channelId];
+    
+
+    function _handleBlockFraudProofs(
+       
+    ) internal {
+      
+       
+
+    }
+
+    function _handleDisputeFraudProofs(
+        // core specific checks
+        // apply proofs
+        // apply dispute to latest state
+        // set state
+    ) internal {
+
+    }
+
+    function _handleTimeoutDispute(
+
```

Comment:
Not sure where you're going with this '_handleBlockFraudProofs', '_handleDisputeFraudProofs', '_handleTimeoutDispute' - the pipeline is the same for everything regardless of the fraud proof (each fraud proof caries its specific logic).

Reply:
each fraud proof carries its own specific logic, exactly thts why I have bundled them together for easier reading and separation